app/default/routes.py

```diff
@@ -158,6 +159,7 @@ def new():
             "account_id": account_id,
             "round_id": request.form["round_id"] or Config.DEFAULT_ROUND_ID,
             "fund_id": request.form["fund_id"] or Config.DEFAULT_FUND_ID,
+            "language": get_lang() or Config.DEFAULT_LANGUAGE,
```

Comment:
Don't think this will be necessary, `get_lang()` will always return something, my concern was more around fetching the the language of existing applications created prior to the migration.

Reply:
have removed the default lang envvar 👍🏾